src/crimson/os/seastore/seastore.cc

```diff
@@ -955,9 +955,10 @@ seastar::future<std::map<uint64_t, uint64_t>> SeaStore::fiemap(
       std::min(size - off, len);
     return _fiemap(t, onode, off, adjust_len);
   }).handle_error(
-    crimson::ct_error::assert_all{
-      "Invalid error in SeaStore::fiemap"
-  });
+    crimson::ct_error::enoent::handle([&oid, FNAME] {
+      INFO("fiemap: object {} doesn't exist, return empty fiemap", oid);
+      return seastar::make_ready_future<std::map<uint64_t, uint64_t>>();
```

Comment:
If the object exists, we return an empty map.  Alienstore does the same thing.  I don't think this makes sense, I think we should be returning an error future as we do in AlienStore::omap_get_header in both cases.

Reply:
done.